### ISSUE TITLE
Convert cleaning script to Python

### ISSUE THREAD

**Issue by newswim** [open-austin/indigent-defense-stats#79]
We have all of the inputs and outputs from the previous correspondence with the Uni of Michigan, so we should be able to build a dictionary which can be used in a function as part of the cleaning process.

This function's type signature would look something like:

```haskell
normalizeStatute :: dirtyStatute -> cleanStatute
```

This workflow includes:

1. Normalizing statute information
2. Other things (@normaljosh can you describe the other parts here?)

**Comment by nicolassaw** [comment]
I've adapted the cleaning script and uploaded it [here](https://github.com/open-austin/indigent-defense-stats/pull/85). I looked through the R code and couldn't find anything else that needed to be added. There was some other text parsing but some of it seemed to be reading from tables I don't know about (and we want to keep it in JSON).

That being said, I think this can be closed unless there is anything else that needs to be updated. Any thoughts? @normaljosh
